egs/tedlium/s5_r2/local/run_learn_lex.sh

```diff
@@ -0,0 +1,134 @@
+#! /bin/bash
+#
+# This script demonstrates a lexicon learning recipe, which aims to imrove
+# the pronounciation of abbreviated words in the TED-LIUM lexicon. It assumes
+# the model exp/tri3 already exists. Please see steps/dict/learn_lexicon.sh
+# for explanation of the options. 
+#
+# Copyright 2016  Xiaohui Zhang
+# Apache 2.0
+
+. ./cmd.sh
+. ./path.sh
+
+oov_symbol="<unk>"
+g2p_mdl_dir=
+# The user may have an English g2p model ready.
+ref_dict=data/local/dict
+# The dir which contains the reference lexicon (most probably hand-derived)"
+# we want to expand/improve, and nonsilence_phones.txt,.etc which we need " 
+# for building new dict dirs."
+data=data/train
+# acoustic training data we use to get alternative"
+# pronunciations and collet acoustic evidence."
+min_prob=0.4
```

Comment:
It's the threshold we apply after normalizing the lexicon s.t. the max-pron-prob for each word is 1.0. I've clarified that.